server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -166,6 +167,18 @@ export class AlfredResourcesFactory implements utils.IResourcesFactory<AlfredRes
 
         const tenantManager = new services.TenantManager(authEndpoint);
 
+        const throttleRequestsPerMs = config.get("alfred:throttling:http:requestsPerMs") as number;
```

Comment:
We should add default values to these parameters in case these variables are missing in the config.
Reason: Don't want the throttling functionality to throw errors and affect Alfred flow.